### PR TITLE
fix: Prevent duplication of wallets across profiles created in same session

### PR DESCRIPTION
**Pull request by marc2332**
Closes https://github.com/iotaledger/firefly/issues/8013

### CODE REVIEW

File: packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export function convertOnboardingProfileToPersistedProfile(
 ): IPersistedProfile {
     return {
         ...DEFAULT_PERSISTED_PROFILE_OBJECT,
+        walletPersistedData: {},
```
Comment (cpl121) [inline packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts:11]: Why clean the information of all the wallets that are already persisted in the profile?

**Review by cpl121 — CHANGES_REQUESTED** [review]
I don't understand why remove too `lastUsedWalletId`

File: packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts (hunk 1 above)
Comment (marc2332) [reply packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts:11]: What profile? This is code is for when you create an onboardingWallet, there is nothing persisted

**Comment by marc2332** [comment]
> I don't understand why remove too `lastUsedWalletId`

because of the same reason, this is an onboarding wallet, there is no lastUsedWalletId, it's pointless

File: packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts (hunk 1 above)
Comment (cpl121) [reply packages/shared/lib/contexts/onboarding/helpers/convertOnboardingProfileToPersistedProfile.ts:11]: makes sense :trollface:

**Review by cpl121 — APPROVED** [review]

**Review by begonaalvarezd — APPROVED** [review]

**Pull request merged**